diff --git a/GNUmakefile b/GNUmakefile
index 5528a9f..96e7088 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -31,6 +31,7 @@ endif
 endif
 
 PREFIX?= /usr/local
+MANDIR?= ${PREFIX}/man
 INSTALLDIR= install -d
 INSTALLBIN= install -g bin -o root -m 555
 INSTALLMAN= install -g bin -o root -m 444
@@ -59,7 +60,7 @@ clean-all:	clean clean-depend
 install:	all
 		$(INSTALLDIR) $(DESTDIR)$(PREFIX)/bin
 		$(INSTALLBIN) tmux $(DESTDIR)$(PREFIX)/bin/tmux
-		$(INSTALLDIR) $(DESTDIR)$(PREFIX)/man/man1
-		$(INSTALLMAN) tmux.1 $(DESTDIR)$(PREFIX)/man/man1/tmux.1
+		$(INSTALLDIR) $(DESTDIR)$(MANDIR)/man1
+		$(INSTALLMAN) tmux.1 $(DESTDIR)$(MANDIR)/man1/tmux.1
 
 -include .depend
